edx_exams/apps/core/exam_types.py

```diff
@@ -35,9 +35,35 @@ class TimedExamType(ExamType):
     is_practice = False
 
 
+class PracticeExamType(ExamType):
+    """
+    Properties for practice exam
+    """
+    name = 'practice'
+    description = 'Practice, non-proctored, timed exam'
+    is_proctored = False
+    is_timed = True
+    is_practice = True
+
+
+class OnboardingExamType(ExamType):
```

Comment:
Suggestion: when this is eventually changed, would this need to be changed? Are there any other changes? I like that you included comments on this, but wondering if any additional documentation or a different approach would make the change cleaner from not supporting onboarding exams to supporting.

Reply:
I don't think any thing would need to change here, if we did create an exam of this type it would 'just work' as proctored but we aren't expecting to be setting these up initially or building onboarding specific features yet.

Reply:
That makes sense! Thanks for clarifying.